libcxx/include/variant

```diff
@@ -781,6 +785,9 @@ _LIBCPP_VARIANT_DESTRUCTOR(
     _Trait::_TriviallyAvailable,
     _LIBCPP_HIDE_FROM_ABI _LIBCPP_CONSTEXPR_SINCE_CXX20 ~__dtor() = default,
     _LIBCPP_HIDE_FROM_ABI _LIBCPP_CONSTEXPR_SINCE_CXX20 void __destroy() noexcept {
+#if __has_feature(memory_sanitizer)
```

Comment:
Actually, now I'm wondering how to get this to work for constexpr variant contexts.

Reply:
Ah, nevermind, I was using `is_constant_evaluated` incorrectly.